library/src/main/java/com/github/javiersantos/bottomdialogs/UtilsLibrary.java

```diff
@@ -9,4 +18,29 @@ static int dpToPixels(Context context, int dp) {
         return (int) (dp * scale + 0.5f);
     }
 
+    static Drawable createButtonBackgroundDrawable(@NonNull Context context, int fillColor) {
```

Comment:
Creates positive button background drawable by code instead of XML, since drawable in XML does not support any customizations in the App side.